docs/source/membership/membership.md

```diff
@@ -132,94 +314,52 @@ operation, all organizations on the channel must agree to it. Therefore, the
-
-### MSP Structure
+## MSP Structure
 
 So far, you've seen that the most important element of an MSP are the specification
 of the root or intermediate CAs that are used to establish an actor's or node's
 membership in the respective organization. There are, however, more elements that are
 used in conjunction with these two to assist with membership functions.
 
-![MSP4](./membership.diagram.5.png)
+A local MSP folder contains the following sub-folders:
 
-*The figure above shows how a local MSP is stored on a local filesystem. Even though
-channel MSPs are not physically structured in exactly this way, it's still a helpful
-way to think about them.*
+![MSP6](./membership.diagram.6.png)
 
-As you can see, there are nine elements to an MSP. It's easiest to think of these elements
-in a directory structure, where the MSP name is the root folder name with each
-subfolder representing different elements of an MSP configuration.
+*The figure above shows the subfolders in a local MSP on the file system*
 
-Let's describe these folders in a little more detail and see why they are important.
+* **config.yaml:**  Used to configure the identity classification feature
+  in Fabric by enabling "Node OUs" and defining the accepted roles.
+
+* **admincerts (Deprecated from Fabric v1.4.3 and higher):** This folder
+  contains a list of identities that define the actors who have the
+  role of administrators for this organization. In general,
+  there should be one or more X.509 certificates in this list.
+
+  **Note:** Prior to Fabric v1.4.3, admins were defined by explicitly putting certs in the
+  `admincerts` folder in the local MSP directory of your peer. **With Fabric v1.4.3 or higher,
+  certificates in this folder are no longer required.** Instead, it is recommended that when
+  the user is registered with the CA, that the `admin` Role is used to designate the node
+  administrator. Then, the identity is recognized as an `admin` by the Node OU value
+  in their signcert. As a reminder, in order to leverage the admin role, the "identity
+  classification" feature must be enabled in the config.yaml above by setting "Node OUs"
+  to `Enable: true`.
+
+  It's worth noting that just because an actor has the role of an administrator it doesn't
+  mean that they can administer particular resources! The actual power a given identity has
+  with respect to administering the system is determined by the policies that manage system
+  resources. For example, a channel policy might specify that `ORG1-MANUFACTURING`
+  administrators have the rights to add new organizations to the channel, whereas the
+  `ORG1-DISTRIBUTION` administrators have no such rights.
 
-* **Root CAs:** This folder contains a list of self-signed X.509 certificates of
+* **cacerts:** This folder contains a list of self-signed X.509 certificates of
   the Root CAs trusted by the organization represented by this MSP.
```

Comment:
Rather than calling out that certificates use the x.509 format, maybe just call it out once somewhere and just say "certificates" thereafter. Also, are root CA's necessarily self signed? Genuine question. And if so, is calling that out really necessary?

Reply:
From the Fabric CA users guide:
The fabric-ca server init command generates a self-signed X.509 certificate. 
And is part of the CA server initialization process. 
After that it issues the certs.

I removed several of the X.509 references where they were redundant.

Reply:
From the Fabric CA users guide:
The fabric-ca server init command generates a self-signed X.509 certificate.
And is part of the CA server initialization process.
After that it issues the certs.

I removed several of the X.509 references where they were redundant.